### PR TITLE
Allows Sechailer Phrases to be Selectable

### PR DESCRIPTION
**Pull request by Ty-Omaha**
Huge thanks to monster, tiger and dzd for helping me out with this and making the code not look like /complete/ potato code.

Ripped from #5336 but less half-assed

Allows you to select the phrase the sechailer will shout at someone with a UI button at the top.

Phrases are broken down into different aggressiveness level groups so you do not have to scroll through 18 phrases to get the one you want. (Unless you set it to the 4th position)

Using a screwdriver on it changes the aggressiveness level.

Using wirecutters on it breaks it and locks it to the current phrase.

99c67e92f7799369d7b8d0510c327fec

46186a065d8f5bd94ecf5706959166d5

:cl:
tweak: Allows sechailer phrases to be selectable
/:cl:

### CODE REVIEW

File: code/datums/action.dm [hunk 1]
```diff
@@ -213,6 +213,10 @@
 /datum/action/item_action/halt
 	name = "HALT!"
 
+/datum/action/item_action/selectphrase
+	name = "Change Phrase"
+	background_icon_state = "bg_revenant"
```
Comment (tigercat2000) [inline code/datums/action.dm]: don't use a different background for standard item actions please

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -190,72 +190,212 @@
...
+					if(17)
+						to_chat(user, "<span class='notice'>You set the restrictor to: I am, the LAW!</span>")
+						phrase = 18
+					if(18)
+						to_chat(user, "<span class='notice'>You set the restrictor to: HALT! HALT! HALT! HALT!</span>")
+						phrase = 1
+			else
+				to_chat(user, "<span class='notice'>It's broken.</span>")
+	else
+		return
```
Comment (tigercat2000) [inline code/modules/clothing/masks/gasmask.dm]: adjust mask will no longer function correctly

File: code/modules/clothing/masks/gasmask.dm [hunk 3]
```diff
@@ -190,72 +190,212 @@
...
-	actions_types = list(/datum/action/item_action/halt)
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/warden
 	name = "\improper Warden SWAT mask"
 	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000. It has a blue stripe."
 	icon_state = "wardenmask"
-	aggressiveness = 3
-	actions_types = list(/datum/action/item_action/halt)
+	aggressiveness = 2
```
Comment (tigercat2000) [inline code/modules/clothing/masks/gasmask.dm]: why did you change this?

File: code/modules/clothing/masks/gasmask.dm [hunk 4]
```diff
@@ -190,72 +190,212 @@
...
+	phrase = 7
+	actions_types = list(/datum/action/item_action/halt,/datum/action/item_action/selectphrase)
```
Comment (tigercat2000) [inline code/modules/clothing/masks/gasmask.dm]: spaces after commas

File: code/datums/action.dm (hunk 1 above)
Comment (Ty-Omaha) [inline code/datums/action.dm]: but wouldn't it look too confusing with the same exact icon 3 times?

Comment (tigercat2000) [inline code/datums/action.dm]: that's why tooltips are a thing

**Comment by Vivalas** [comment]
I can't wait to always set mine to "You have the right to shut the fuck up."

**Comment by Ty-Omaha** [comment]
My favorite is "face the wrath of the golden bolt"

File: code/datums/action.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/datums/action.dm]: Still need to change the background.

File: code/modules/clothing/masks/gasmask.dm [hunk 5]
```diff
@@ -190,72 +190,210 @@
...
+	var/aggressiveness = 1
 	var/safety = 1
-	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust)
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/hos
 	name = "\improper HOS SWAT mask"
 	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000. It has a tan stripe."
 	icon_state = "hosmask"
 	aggressiveness = 3
```
Comment (Fox-McCloud) [inline code/modules/clothing/masks/gasmask.dm]: needs phrase set to 12

Comment (Ty-Omaha) [inline code/modules/clothing/masks/gasmask.dm]: fixed

File: code/modules/clothing/masks/gasmask.dm [hunk 6]
```diff
@@ -190,72 +190,213 @@
...
-	actions_types = list(/datum/action/item_action/halt)
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/ui_action_click(mob/user, actiontype)
 	if(actiontype == /datum/action/item_action/halt)
 		halt()
-	else
+	if(actiontype == /datum/action/item_action/adjust)
 		adjustmask(user)
+	if(actiontype == /datum/action/item_action/selectphrase)
```
Comment (TheDZD) [inline code/modules/clothing/masks/gasmask.dm]: The lines `if(actiontype == /datum/action/item_action/adjust)` and `if(actiontype == /datum/action/item_action/selectphrase)` should be changed to `else if` rather than just `if`, as otherwise you'll needlessly check these despite the fact that they're never going to evaluate to `true`.
